vignettes/HOWTO-BCV.rmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "HOWTO generate biocViews HTML"
+author: "S. Falcon and V.J. Carey"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Quick-start Guide}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document:
+    number_sections: no
+    toc: yes
+    toc_depth: 2
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+
+```{r include=FALSE}
```

Comment:
I think rather than `include=FALSE`, we should do `echo=FALSE`. Also remove the surrounding spaces inside the 3 tick marks for this and other instances.